quickstarts/uppercase/functions/package.json

```diff
@@ -12,8 +12,9 @@
     "sinon": "^4.1.3"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha --reporter spec",
-    "lint": "./node_modules/.bin/eslint .",
+    "build": "npm install",
```

Comment:
@nicolasgarnier got the tests running by adding this `build` step.

Reply:
build should not do NPM install; that takes way too long and is needed so infrequently. It should be part of our travis rules.

Reply:
What I want is for "test" do depend on "install" so that when I do `lerna run test` it will run `npm install` in only the directories that have tests.

Reply:
How about I make a `travis-test` target that does `npm install && npm run test`